deploy.sh

```
@@ -0,0 +1,265 @@
+mkdir -p "$BACKUP_DIR"
+
+echo -e "${GREEN}✓${NC} Backup created: $BACKUP_DIR"
+echo ""
+
+# ============================================================================
+# Step 3: Prepare Files for Upload
+# ============================================================================
+echo -e "${YELLOW}📦 Step 3: Preparing Files for Upload${NC}"
+
+# Files/Directories to upload
+UPLOAD_ITEMS=(
```

Shellcheck (0.11.0)

[warning] UPLOAD_ITEMS appears unused. Verify use (or export if used externally).

(SC2034)